src/common/react/named-sfc.ts

```diff
@@ -2,10 +2,10 @@
 // Licensed under the MIT License.
 import * as React from 'react';
 
-export type ReactSFCWithDisplayName<P = {}> = React.SFC<P> & { displayName: string };
+export type ReactFCWithDisplayName<P = {}> = React.FC<P> & { displayName: string };
 
-export function NamedSFC<P = {}>(displayName: string, component: React.SFC<P>): ReactSFCWithDisplayName<P> {
+export function NamedFC<P = {}>(displayName: string, component: React.FC<P>): ReactFCWithDisplayName<P> {
```

Comment:
I think there's a few instances of the string "SFC" in the tests for this that should probably also get updated

Reply:
updated